src/ra_fwd.h

```diff
@@ -43,7 +43,9 @@ using LPVOID = void*;
 #endif // !_WINDEF_
 
 #ifndef _WINNT_
-using HANDLE = void*;
+using HANDLE  = void*;
+using LPCTSTR = const TCHAR*;
+using LPTSTR  = TCHAR* ;
```

Comment:
I moved these here to not obscure the type.
They were previously in the `ra `namespace, now they're in the global namespace.